tools/amd_build/disabled_features.yaml

```diff
@@ -23,16 +23,10 @@
             "HIPBLAS_DATA_HALF": "0"
         }
       },
-      {
-        "path": "aten/src/THCUNN/SoftMaxCommon.cuh",
-        "functions": {
-          "cudaOccupancyMaxActiveBlocksPerMultiprocessor": "16"
-        }
-      },
       {
         "path": "aten/src/ATen/native/cuda/SoftMax.cu",
         "functions": {
-          "cudaOccupancyMaxActiveBlocksPerMultiprocessor": "16"
+          "cudaOccupancyMaxActiveBlocksPerMultiprocessor": "max_active_blocks = 16;"
```

Comment:
I'm working on the disabled_features right now (see #30 ). I'm #ifdef ing this in the source. We can discuss what is preferable in #30. Please remove changes to disabled_features here.